### PR TITLE
llnode 1.7.0

### PR DESCRIPTION
**Pull request by mistydemeo**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

This is kind of a mess, since it dropped its gyp shim and now depends on the node-gyp NPM package being installed.

cc @ilovezfs

### CODE REVIEW

File: Formula/llnode.rb [hunk 1]
```diff
@@ -11,14 +13,10 @@ class Llnode < Formula
     sha256 "e047f606e4923900a3285acc8a352f847d387e877d5436ddf5cdbf37d256cf27" => :el_capitan
   end
 
-  depends_on "python@2" => :build
+  depends_on "node" => :build
+  depends_on "python@2" => :build if MacOS.version <= :snow_leopard
```
Comment (ilovezfs) [inline Formula/llnode.rb]: This can just be `depends_on "python@2" => :build`. The `if snow leopard` thing has been removed everywhere

Comment (mistydemeo) [reply Formula/llnode.rb]: Good catch, accidentally caught this in a rebase.

**Comment by ilovezfs** [comment]
Does the `node-gyp` that comes with the node formula itself not work?
/usr/local/Cellar/node/10.2.1/libexec/lib/node_modules/npm/node_modules/node-gyp

**Comment by ilovezfs** [comment]
Neat! Thanks for fixing this :)

**Pull request closed without merging**